api/appendices/resource_attributes.adoc

```diff
@@ -74,6 +75,33 @@ POST /api/rates
 
 ___
 
+[[groups]]
+=== Group Attributes
+
+----
+POST /api/groups
+----
+
+==== Required attributes:
+
+[cols="2,1,3",options="header",width="100%"]
+|=====================
+| Attribute | Type | Description
+| description | string | Description
+| role | hash | Role the group is assigned to, requires an *href*, *id* or *name* attribute in the hash
```

Comment:
Have we used "hash" everywhere? I wonder if it's better to use "object" to describe these, as it's JSON?